digital-twin/device/src/main/java/com/microsoft/azure/sdk/iot/digitaltwin/device/AbstractDigitalTwinComponent.java

```diff
@@ -36,11 +35,11 @@
     public static final int STATUS_CODE_PENDING = 202;
     /** Indicates the content or operation is successfully completed. */
     public static final int STATUS_CODE_COMPLETED = 200;
-    static final String COMMAND_NOT_IMPLEMENTED_MESSAGE_PATTERN = "\"Command is not implemented for interface instance [%s].\"";
+    static final String COMMAND_NOT_IMPLEMENTED_MESSAGE_PATTERN = "\"Command is not implemented for component [%s].\"";
     /**
-     * The interface instance name associated with this interface Id. For example, environmentalSensor.
+     * The component name associated with this interface Id. For example, environmentalSensor.
      */
-    private final String digitalTwinInterfaceInstanceName;
+    private final String digitalTwinComponentName;
     /**
      * The interfaceId of interface to be registered.  For example, urn:contoso:com:EnvironmentalSensor:1.
```

Comment:
Are we changing all instances of the word "interface" to "component". If so, there are others.